src/data/actions/assets.js

```diff
@@ -319,6 +319,11 @@ export const uploadExceedMaxSize = maxFileSizeMB => ({
   maxFileSizeMB,
 });
 
+export const uploadInvalidFileType = validFileTypes => ({
```

Comment:
I'm not sure I understand what validFileTypes does here. Is it intended for something later?

Reply:
Ah I was passing this through for this logic: https://github.com/edx/studio-frontend/blob/51a300ee1f766a94e7c5eaedbc11d52eae8f7b8b/src/components/AssetsDropZone/index.jsx#L27 

`else if (rejectedFiles.length > 0) {
      // bubbling up error from the first rejected file in the list
      const rejectedFile = rejectedFiles[0];
      if (rejectedFile.size > this.dropZoneMaxFileSizeBytes) {
        this.props.uploadExceedMaxSize(this.dropZoneMaxFileSizeBytes);
      } else {
        this.props.uploadInvalidFileType(this.props.acceptedFileTypes);
      }
    }`

The `else` logic. I was hoping to make that an `else if` to be more specific, but I think to do that effectively we'll need to import another package (as the dropzone uses an external package for that logic) and I ended up with the plain else instead. I'll go through and clean this up for now as well. If we decide to make that more explicit we can.